content/sifixes/src/npcs/neyobi.uc

```diff
@@ -0,0 +1,92 @@
+/*  Copyright (C) 2016  The Exult Team
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Neyobi is a Gwani child. She is asleep with an illness that can only
+ *  be cured with blood from an ice dragon. The original code for her is
+ *  incomplete, so while she can be cured, she always goes back to her bed.
+ *  This file gives her a schedule so she moves around as her existing
+ *  dialog suggests.
+ *
+ *  2016-07-09 Written by Knight Captain
+ */
+ 
+void Neyobi object#(0x493) () 
+{
+	if (event == DOUBLECLICK)
+	{
+		if (NEYOBI->get_item_flag(SI_ZOMBIE)) // If she hasn't been cured yet.
+            {
+            AVATAR->item_say("@Wake up, little one!@"); // Only works if partyUtters is commented out or placed before this line. Bug 1957.
+            partyUtters(1, "@She cannot wake, Avatar. She is very sick.@", "@Poor little one! She is very sick.@", false);
+            }
+        if (!NEYOBI->get_item_flag(SI_ZOMBIE)) // If she is no longer sick.
+            {
+            AVATAR->item_say("@Good morning, little one!@");
+	    delayedBark(NEYOBI, "@Tee hee hee hee!@", 3); // Per original usecode. Does not appear, Bug 1957.
+            // This next if section is a hack to get around having to edit the NPC on the map.
+            // Otherwise we would only need to take her out of the WAIT activity.
+            if (!NEYOBI->get_item_flag(MET)) // If we haven't met her yet, give her a schedule.
+                {        
+                NEYOBI->set_new_schedules(MIDNIGHT, WANDER, [0x408, 0x3A4]); // Outside of family cave.
+                NEYOBI->modify_schedule(MORNING, SLEEP, [0x3F5, 0x36A]); // Similar to parents, Yenani and Myauri.
+                NEYOBI->modify_schedule(AFTERNOON, LOITER, [0x3EC, 0x359]); // No place to sit and eat. 
+                NEYOBI->modify_schedule(EVENING, WANDER, [0x3B6, 0x35E]); // With Yenani to learn.
+                NEYOBI->modify_schedule(NIGHT, LOITER, [0x473, 0x350]); // With Baiyanda, should not step in fire.
+                // set_new_schedules can only have a single one in its clause, as it overwrites everything else.
+                // In this case we use it first to clear Neyobi's existing WAIT at DAWN, then fill in the rest with
+                // modify schedule, which only updates that daypart. Not sure how to clear a single entry otherwise.
+                }
```

Comment:
I had a hard time trying to get the tabs for this correct, so if you can get it the way you'd like, please do so.